tests/Aspire.Dashboard.Components.Tests/UseCultureAttribute.cs

```diff
@@ -63,7 +62,7 @@ public UseCultureAttribute(string culture, string uiCulture)
     /// and replaces them with the new cultures defined in the constructor.
     /// </summary>
     /// <param name="methodUnderTest">The method under test</param>
-    public override void Before(MethodInfo methodUnderTest)
+    public override void Before(MethodInfo methodUnderTest, IXunitTest test)
```

Comment:
What are these for? Also we should probably add tripple slash comments to the new param.

Reply:
It contains extra information such as timeout, test method arguments, whether it's an explicit test, etc. We don't need any info about the test for the purpose of `[UseCulture]`. We didn't previously needed any info from `methodUnderTest` as well :)